rllib/agents/maml/maml_torch_policy.py

```diff
@@ -0,0 +1,432 @@
+import logging
```

Comment:
Same comment as in the other PR: Two tests need to be added: 1) add torch to `framework_iterator` loop in test_maml.py 2) Add 2 regression tests (tf and torch) to tuned_examples and activate them in `rllib/BUILD`.

Reply:
The Pytorch commit is supposed to be separate and in the other PR...Ill procede to fix this branch...

Reply:
@sven1977 I think tests are good to go now... also completed Eric's requested changes